### PR TITLE
Feature/svs sensor setup

### PR DESCRIPTION
**Pull request by Dtananaev**

### CODE REVIEW

File: start_drive_tsr.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
     CTRL + R     : toggle recording of simulation (replacing any previous)
     CTRL + P     : start replaying last recorded simulation
-    CTRL + +     : increments the start time of the replay by 1 second (+SHIFT = 10 seconds)
+    CTRL cd+ +     : increments the start time of the replay by 1 second (+SHIFT = 10 seconds)
```
Comment (Logrus) [inline start_drive_tsr.py]: accidental change?

File: svs_only_tsr.py [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+        self.height=height
+        self.fov=fov
+        self.tick=tick
+        self._pygame_disp=pygame_disp #display for pygame
+        self._surface = None # image to render fom array for pygame display
+        self.camera = None
+
+    def add_camera(self, x=-6.5, y=0.0, z=2.7, 
+                   roll=0, pitch=0, yaw=0,
+                   camera_type='sensor.camera.rgb',
```
Comment (Logrus) [inline svs_only_tsr.py]: Is it standard pinhole projection so far?

**Review by Logrus — APPROVED** [review]

File: svs_only_tsr.py [hunk 3]
```diff
@@ -0,0 +1,452 @@
...
+            self.destroy()
+            self.player = self.world.try_spawn_actor(vehicle_bp, spawn_point)
+        while self.player is None:
+            spawn_points = self.map.get_spawn_points()
+            spawn_point = random.choice(spawn_points) if spawn_points else carla.Transform()
+            self.player = self.world.try_spawn_actor(vehicle_bp, spawn_point)
+
+
+        # Svs front camera
+        self.cam1_svs_front=CameraTsr(vehicle=self.player, width=svs_resolution[0], height=svs_resolution[1], fov="90",pygame_disp=True)
```
Comment (Logrus) [inline svs_only_tsr.py]: Let's use autopep8 to make sure that we comply to PEP8 in our scripts and also do the auto formatting:
https://pypi.org/project/autopep8/

Comment (Logrus) [reply svs_only_tsr.py]: When installed, use ```autopep8 --in-place <filename>``` on this file (don't use --aggressive flag)

File: start_drive_tsr.py (hunk 1 above)
Comment (Dtananaev) [reply start_drive_tsr.py]: Yeas I fixed it and update file the "cd" is typo .

File: svs_only_tsr.py (hunk 2 above)
Comment (Dtananaev) [reply svs_only_tsr.py]: Yes

File: svs_only_tsr.py (hunk 3 above)
Comment (Dtananaev) [reply svs_only_tsr.py]: Thanks I will do that:)

**Comment by Logrus** [comment]
Great! I think this one is good to be merged :+1:

**Pull request merged**